.github/workflows/on-merge-to-develop.yml

```diff
@@ -0,0 +1,113 @@
+# Inspired from https://github.com/backstage/backstage/blob/master/.github/workflows/ci.yml. Thanks!
```

Comment:
I understand your point and I tried to give it a shot: for long lines and long comments I totally agree with you. But for very short explanations I feel new lines can sometimes on the contrary make the reading harder.
Since this is indeed personal preference and didn't want to take too much time on this, but I'll be vigilant in the future when writing YAML.
Thanks for your feedback